www/src/pages-markdown/whatsnew.md

```diff
@@ -1,33 +1,48 @@
 ---
-title: "Whats New"
+title: "What's New"
 date: "2020-02-01"
 path: "/whats-new"
 header: true
 ---
 
-Thanks for playing this beta of Spellsource!
+## What's New
```

Comment:
It sort of doesn't make sense to make an array using text here, we should just store the array of whats new on a date/version basis